scripts/pyladies-twitter-scraper.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+
+"""
+script to generate PyLadies twitter .csv with the following information:
+- twitter handle
+- description
+- expanded tweet profile url
+- date of last tweet
+
+To use you'll need to register a Twitter app at developer.twtter.com for the follwing values:
+- CLIENT_ID
+- CLIENT_SECRET
+- ACCESS_TOKEN
+- ACCESS_TOKEN_SECRET
+"""
+
+__author__ = "Lorena Mesa"
+__email__ = "lorena@pyladies.com"
+
+import csv
+from datetime import datetime
+
+import json
+
+import requests
+from requests_oauthlib import OAuth1
+
+
+MAX_USER_RESULTS = 1000
+MAX_USER_COUNT = 20
+
+CLIENT_ID = 'client-id'
+CLIENT_SECRET = 'client-secret'
+ACCESS_TOKEN = 'access-token'
+ACCESS_TOKEN_SECRET = 'access-token-secret'
+
+PYLADIES_LOCATIONS_LIST = 'pyladies-locations'
+
+
+def build_request(url, search_params):
+	twitter_api = 'https://api.twitter.com/1.1'
+
+	request = requests.get(
+		url=f'{twitter_api}/{url}', 
+		auth=OAuth1(
+			CLIENT_ID,
+			CLIENT_SECRET,
+			ACCESS_TOKEN,
+			ACCESS_TOKEN_SECRET
+		),
+		params=search_params
+	)
+	return request
+
+def make_request(num_requests, url, search_params, page_number, key):
+	data = []
+
+	for i in range(1, num_requests+1):
+		if page_number:
+			search_params['page'] = f'{i}'
+		
+		response = build_request(url=url,search_params=search_params)   
+
+		if response.ok:
+			response = response.json()
+			if key:
+				data += response.get(key)
+			elif isinstance(response, dict):
+				data.append(response)
+			else:
+				data += response
+	return data
+
+
+# Get PyLadies twitter handles data from Twitter API
+print('Getting known PyLadies Locations list members...')
+pyladies_lists = make_request(
+	num_requests=1,
+	url='lists/list.json',
+	search_params={'screen_name': 'pyladies'},
+	page_number=False,
+	key=None
+)
+
+pyladies_lists = [a_list.get('id') for a_list in pyladies_lists if PYLADIES_LOCATIONS_LIST in a_list.get('full_name')]
+
+known_pyladies_handles = make_request(
+	num_requests=1,
+	url='lists/members.json',
+	search_params={'list_id': f'{pyladies_lists[0]}', 'skip_status': True, 'count': 5000},
+	page_number=False,
+	key='users'
+)
+
+print('Done getting known PyLadies Locations handles...')
+
+print('Searching...')
+pyladies_handles = make_request(
+	num_requests=int(MAX_USER_RESULTS/MAX_USER_COUNT),
```

Comment:
Since the two MAX variables are integers, you could avoid the cast to int by using floor division `MAX_USER_RESULTS // MAX_USER_COUNT`